src/validations.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-misleading-character-class */
+'use strict'
+// https://v8.dev/features/regexp-v-flag
+// https://mathiasbynens.be/notes/es-unicode-property-escapes
+// https://www.regular-expressions.info/unicode.html
+
+const CHARACTERS = '\\p{L}\\p{M}'
+const INPUT_BASE_REG_EXP = new RegExp(`-0-9.,;:<>?/\n${CHARACTERS}'_ &()/+@%`, 'u')
+const DIGITS = '0-9'
+
+const REGEX_VALIDATION = {
+  NUMBER: new RegExp(`^[${DIGITS}]+$`),
+  EMAIL: /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/,
```

Comment:
I think this regex fix better on rfc2822
```suggestion
  EMAIL: /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$/i,
```

Reply:
This is updated per suggestions.